src/docs/feature-flags/index.mdx

```diff
@@ -46,41 +48,36 @@ we'll build a feature called `test-feature` scoped at the _organization_ level
 ### Determine the permanency of your feature
 
 Typically we use feature flags for development. They are usually [intended to
-be graduated](#after-launch-graduation). Feature flag that controls a
-subscription-based feature for sentry.io are permanent.
+be graduated](#after-launch-graduation). The only exceptions to this are permanent
+flags which control [subscription plan-specific features](#plan-specific-features)
+in GetSentry.
 
-Most feature flags are placed in `temporary.py`, permanent flags live in
+Most Sentry feature flags are placed in `temporary.py`, while permanent Sentry flags live in
 `permanent.py`.
 
-### Add your feature to server.py
+GetSentry only flags are typically placed in [`features.py`](https://github.com/getsentry/getsentry/blob/master/getsentry/features.py).
 
-[`conf/server.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py)
-contains many of the default settings in the application. Here you will add
-your feature, and decide what default value it should hold unless specified by
-the user.
+### Add your feature to the FeatureManager
 
-The `SENTRY_FEATURES` dictionary contains all the features in the application
-with their corresponding scope. Your feature should start off disabled by default:
+If you want to back your feature flag via options, you can do so using the [Flagpole](/feature-flags/flagpole/) library
+by adding the feature to the `FeatureManager` with the `FLAGPOLE` enum set as the feature strategy:
 
 ```python
-SENTRY_FEATURES = {
-    'organizations:test-feature': False,
-    'auth:register': True,
-    # ...
-    'projects:minidump': False,
-}
+default_manager.add('organizations:test-feature', OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE)
```

Comment:
Once we no longer have flagr, I think the strategy enum because less useful and is something we should explore removing to simplify feature flagging further.